eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -44,22 +44,6 @@ import scala.util.{Failure, Success, Try}
  */
 
 object Helpers {
-
-  /**
-   * Depending on the state, returns the current temporaryChannelId or channelId
-   *
-   * @return the long identifier of the channel
-   */
-  def getChannelId(stateData: Data): ByteVector32 = stateData match {
```

Comment:
what about this transfer of `channelId` from special method to `Data` perameter overridden where appropriate? Seems to make a code a bit clearer in `Channel`.

Reply:
I like it, it's better than what we had before :+1: IMO